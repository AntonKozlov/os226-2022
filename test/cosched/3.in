# 	entryid	cnt	prio	deadline
coapp	2	2	0	-1
coapp	2	2	0	2
coapp	2	2	0	-1

cosched 2
