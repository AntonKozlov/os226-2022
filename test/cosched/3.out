        coapp_rt id 2 cnt 2
        coapp_rt id 2 cnt 1
        coapp_rt id 2 cnt 0
        coapp_rt id 1 cnt 2
        coapp_rt id 3 cnt 2
        coapp_rt id 1 cnt 1
        coapp_rt id 3 cnt 1
        coapp_rt id 1 cnt 0
        coapp_rt id 3 cnt 0
